src/main/java/com/epam/paymentsystem/service/repository/impl/BankAccountRepositoryImpl.java

```diff
@@ -0,0 +1,48 @@
+package com.epam.paymentsystem.service.repository.impl;
+
+import com.epam.paymentsystem.service.model.BankAccount;
+import com.epam.paymentsystem.service.model.enums.Status;
+import com.epam.paymentsystem.service.repository.BankAccountRepository;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class BankAccountRepositoryImpl implements BankAccountRepository {
+    private final List<BankAccount> list = new ArrayList<>();
+
+    @Override
+    public BankAccount getBankAccount(String bankAccountID) {
+        return list.stream()
+                .filter(
+                        bankAccount -> String.valueOf(bankAccount.getAccountID()).equals(bankAccountID))
+                .findFirst()
+                .orElseThrow(() -> new RuntimeException("BankAccount is not found!"));
```

Comment:
I would recommend to create specific custom exception, like BankAccountNotFound or EntityNotFound instead of RuntimeException